Makefile

```diff
@@ -212,6 +213,14 @@ $(FLAKE8): $(PYTEST)
 test-deps: $(PY)
 	$(call PIP,test-requirements.txt)
 
+$(SELENIUM): $(PY)
+	@# Because shelltoolbox requires ez_setup already installed before being
+	@# installed we need to manually do them this way instead of via the
+	@# requirements.txt.
+	bin/pip install --no-index --no-dependencies --find-links $(WHEEL_CACHE) --find-links $(PYTHON_CACHE) ez_setup==0.9
+	bin/pip install --no-index --no-dependencies --find-links $(WHEEL_CACHE) --find-links $(PYTHON_CACHE) shelltoolbox==0.2.1
```

Comment:
This is a good idea, I'll bring it up with JC tomorrow to see if this can be expanded on.